### PR TITLE
Add vault strategy contribution disclaimer

### LINKED ISSUE #106: Include "Strategy rewards" disclaimer on Vaults page
Improvement suggestion:

Include a "Strategy rewards" disclaimer at the bottom of the vaults page that explain how we reward strategists. A call to action for people to write strategies for Yearn per se.

[Example on NEAR protocol website](https://twitter.com/thibauld/status/1316182347943944192?s=21)

EkQEnfFXcAAKHrY

### PR DESCRIPTION
**Pull request by wirjo**
Closes #106 

Footer disclaimer on vaults page to link to the new doc
image

### CODE REVIEW

**Comment by milkyklim** [comment]
@wirjo need exact numbers here, see FAQ for info: https://docs.yearn.finance/faq#what-are-the-fees

**Comment by wirjo** [comment]
1/ Clarified number in footer disclaimer: 
image

**Comment by wirjo** [comment]
2/ Corresponding docs also updated with clarification: 

image

File: src/components/vault/vault.jsx [hunk 1]
```diff
@@ -666,6 +677,29 @@ class Vault extends Component {
...
+              <Grid item><TimelineIcon fontsize="small" /></Grid>
+              <Grid item xs>
+                <Typography variant="h4" style={{ display: 'inline', fontWeight: 'bold' }}>
+                Strategy Rewards
+                </Typography>
+              </Grid>
+            </Grid>
+
+            <Typography variant="body1">
+              Yearn is only possible due to community contributions. Vault strategy contributors are rewarded with <b>0.5%</b> of yield generated from by the vault. <a href="https://docs.yearn.finance/how-to-guides/how-to-contribute-a-yvault-strategy/" target="_blank">Learn more &rarr;</a>
```
Comment (x48-crypto) [inline src/components/vault/vault.jsx:697]: Where did we land on this @wirjo ? Still pending change?

Comment (wirjo) [reply src/components/vault/vault.jsx:697]: I think this is fine now but maybe @milkyklim should just make a call and merge this.

Comment (milkyklim) [reply src/components/vault/vault.jsx:697]: Please, merge if code it fine @x48-crypto.

**Pull request merged**